uptick/chain.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "uptick",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://uptick.network/",
+  "pretty_name": "Uptick",
+  "chain_id": "uptick_117-1",
+  "bech32_prefix": "uptick",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md